src/util/util_errors.h

```diff
@@ -137,6 +137,7 @@ enum sssd_errors {
     ERR_SSL_FAILURE,
     ERR_UNABLE_TO_VERIFY_PEER,
     ERR_UNABLE_TO_RESOLVE_HOST,
+    ERR_UID_OUTSIDE_RANGE,
```

Comment:
As you added a new error in util_errors.h, please, also to add a new string for this error on util_errors.c